services/ocs/pkg/config/config.go

```diff
@@ -24,5 +24,26 @@ type Config struct {
 
 	TokenManager *TokenManager `yaml:"token_manager"`
 
+	RevaGateway string `yaml:"reva_gateway" env:"OCIS_REVA_GATEWAY;REVA_GATEWAY" desc:"CS3 gateway used to look up user metadata" deprecationVersion:"3.0" removalVersion:"4.0.0" deprecationInfo:"REVA_GATEWAY changing name for consistency" deprecationReplacement:"OCIS_REVA_GATEWAY"`
```

Comment:
Why do we have this here, I thought that the deprecations have been fully cleared up??

Reply:
Branch created before the deprecation. Removed it